docusaurus.config.js

```diff
@@ -141,17 +145,17 @@ const config = {
             ],
           },
         ],
-        logo: {
-            alt: 'Apache Incubator logo',
-            src: 'img/apache-incubator.svg',
-            href: 'https://incubator.apache.org/'
-        },
-        copyright: `<div style="text-align: left;">
-          <div>
-            <p style="font-family: Avenir-Medium,serif;font-size: 14px;color: #999;line-height: 20px;"> Apache Kvrocks is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF. </p>
-          </div>
-          <div style="border-top: 1px solid #ccc;min-height: 60px;line-height: 20px;text-align: center;font-family: Avenir-Medium,serif;font-size: 14px;color: #999;display: flex;align-items: center;"><span>Copyright © ${new Date().getFullYear()} The Apache Software Foundation. Apache Kvrocks, Kvrocks, and its feather logo are trademarks of The Apache Software Foundation.</span></div>
-        </div>`,
+        // logo: {
+        //     alt: 'Apache Incubator logo',
+        //     src: 'img/apache-incubator.svg',
+        //     href: 'https://incubator.apache.org/'
+        // },
+        // copyright: `<div style="text-align: left;">
+        //   <div>
+        //     <p style="font-family: Avenir-Medium,serif;font-size: 14px;color: #999;line-height: 20px;"> Apache Kvrocks is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF. </p>
+        //   </div>
+        //   <div style="border-top: 1px solid #ccc;min-height: 60px;line-height: 20px;text-align: center;font-family: Avenir-Medium,serif;font-size: 14px;color: #999;display: flex;align-items: center;"><span>Copyright © ${new Date().getFullYear()} The Apache Software Foundation. Apache Kvrocks, Kvrocks, and its feather logo are trademarks of The Apache Software Foundation.</span></div>
+        // </div>`,
```

Comment:
As mentioned above, these need to be kept as the site belongs to ASF.

Reply:
Sorry, I blindly modified according to the UI and ignored ASF. I'll add back in my next commit.